src/odemis/gui/comp/combo.py

```diff
@@ -167,3 +175,76 @@ def draw(self, dc):
         back_brush = wx.Brush(back_colour, wx.BRUSHSTYLE_SOLID)
         dc.SetBackground(back_brush)
         dc.Clear()
+
+        
+ITEM_WIDTH = 200
+ITEM_HEIGHT = 28
+COLOBAR_WITH_RATIO = 0.5
+
+
+class ColorMapComboBox(ComboBox):
+    """
+    A special combo box that is designed for allowing the user to select a color map
+    """
+
+    def __init__(self, *args, **kwargs):
+        """
+        labels: list of str that are the names of color maps
+        choices: list of matplotlib colormap. List of corresponding color map objects
+        """
+        labels = kwargs['labels']
+        colormaps = kwargs['choices']
+        self.choices = labels
```

Comment:
I don't think you need this, instead use `.Strings` or `.GetItems()`. Also, it's confusing to put the content of `labels` in `.choices`!